src/plugins/irc-events/mode.js

```diff
@@ -6,6 +6,29 @@ var Msg = require("../../models/msg");
 
 module.exports = function(irc, network) {
 	var client = this;
+
+	irc.on("channel info", function(data) {
+		if (!data.modes) {
+			return;
+		}
+
+		const targetChan = network.getChannel(data.channel);
+		if (typeof targetChan === "undefined") {
+			return;
+		}
+
+		for (var i = 0; i < data.modes.length; i++) {
+			var mode = data.modes[i];
```

Comment:
This is a minor nitpick, but any reason why not to use `data.modes.forEach(mode => { ... });` instead?
Also, `const` when possible?